src/main/java/com/synopsys/integration/alert/web/controller/handler/SystemHandler.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * blackduck-alert
+ *
+ * Copyright (C) 2018 Black Duck Software, Inc.
+ * http://www.blackducksoftware.com/
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.web.controller.handler;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+
+import com.synopsys.integration.alert.common.ContentConverter;
+import com.synopsys.integration.alert.web.actions.SystemActions;
+import com.synopsys.integration.alert.web.model.SystemMessageModel;
+
+@Component
+public class SystemHandler extends ControllerHandler {
+    private final SystemActions actions;
+
+    @Autowired
+    public SystemHandler(final ContentConverter contentConverter, final SystemActions actions) {
+        super(contentConverter);
+        this.actions = actions;
+    }
+
```

Comment:
I would like to see three methods here:
1. A getter that takes a start date and end date
2. A getter to get the latest (current date - a day or week)
3. A getter for all

Reply:
The controller should follow suit.